.circleci/config.yml

```diff
@@ -141,11 +141,23 @@ workflows:
       - test:
           requires: [build]
           matrix:
+            alias: test-no-tl
             parameters:
-              suite: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+              suite: [0, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 13]
           filters:
             tags:
               only: /.*/
+      - test:
+          requires: [build]
+          matrix:
+            alias: test-tl
+            parameters:
+              suite: [8]
+          filters:
+            branches:
+              only: /(tlmete.*|develop)/
+            tags:
+              only: /.*/
```

Comment:
This is a crude solution and we still need to wait 8 minutes extra when releasing RCs (that can at least be changed on the spot by a POC if needed and if they are aware)